src/accordion/accordion.style.tsx

```diff
@@ -15,13 +15,23 @@ export const Content = styled.div`
 
 export const TitleWrapper = styled.div<TitleWrapperStyleProps>`
     display: flex;
-    flex-direction: ${(props) => (props.$hasTitle ? "row" : "column")};
-    align-items: ${(props) => (props.$hasTitle ? "center" : "flex-end")};
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
     padding-bottom: 1rem;
 
     ${MediaQuery.MaxWidth.mobileL} {
         justify-content: flex-end;
     }
+
+    ${(props) => {
+        if (!props.$showTitleInMobile && !props.$hasExpandAll) {
+            return css`
+                display: none;
+                visibility: hidden;
+            `;
```

Comment:
- this should be wrapped in mobileL media query
- can remove visibility style, I dont see how it has any effects

Reply:
updated